fml/logging.h

```diff
@@ -27,7 +27,10 @@ class LogMessage {
 
   std::ostream& stream() { return stream_; }
 
+  static void CaptureNextLog(const std::shared_ptr<std::ostringstream>& stream);
```

Comment:
This should probably be a free function in `fml` right?  The relationship between FML_LOG and LogMessage isn't clear.